component-models.json

```diff
@@ -1914,5 +1914,41 @@
         "value": ""
       }
     ]
+  },
+  {
+    "id": "search",
+    "fields": [
+      {
+        "component": "text-input",
+        "valueType": "string",
+        "name": "searchPlaceholderText",
+        "label": "Search Placeholder Text"
+      },
+      {
+        "component": "richtext",
```

Comment:
@mayur-satav, for phase 1, let's focus on pulling the suggestions list from page titles. That means when users enter terms, it'll search for suggestions within page titles and display the respective titles. The static suggestion list can be considered for a later phase. 

So you can take this option out.

Reply:
ok